samples/meetings-transcription/csharp/MeetingTranscription/Helpers/GraphHelper.cs

```diff
@@ -0,0 +1,93 @@
+﻿// <copyright file="GraphHelper.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace MeetingTranscription.Helpers
+{
+    using Microsoft.Extensions.Options;
+    using Microsoft.Identity.Client;
+    using MeetingTranscription.Models.Configuration;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+    using System.Net.Http;
+    using MeetingTranscription.Models;
+    using Newtonsoft.Json.Linq;
+
+    public class GraphHelper
+    {
+        /// <summary>
+        /// Stores the Azure configuration values.
+        /// </summary>
+        private readonly IOptions<AzureSettings> azureSettings;
+
+        const string graphBetaEndpoint = "https://graph.microsoft.com/beta";
```

Comment:
Can we read it from appsettings because endpoint is going to change in the near future?